src/lib/useKeyfileGenerator.ts

```diff
@@ -242,14 +253,56 @@ export const useSingleKeyfileGenerator = ({
       revision: point.keyRevisionNumber,
     });
 
-    if (Just.hasInstance(derivedSeed)) {
-      setSeeds([derivedSeed.value]);
+    if (Just.hasInstance(defaultDerivedSeed)) {
+      setDefaultSeeds([defaultDerivedSeed.value]);
+    }
```

Comment:
I think this wants an early-out? We don't want to proceed to fallback logic if we got the default one, right?

```suggestion
      setDefaultSeeds([defaultDerivedSeed.value]);
      return;
    }
```

Reply:
Typically we would, but in this case we want to derive both seeds if possible. The comparison to on-chain keys happens downstream in `useKeyfileGenerator` (which is wrapped by this hook). Will add some comments to reflect this behavior.